crabs/detection_tracking/datamodules.py

```diff
@@ -152,19 +152,19 @@ def prepare_data(self):
         """
         pass
 
-    def setup(self):
+    def setup(self, stage: str):
```

Comment:
Sorry, I was overlooking this. Yes we need stage argument, we just don't need the condition check, sorry. I set to None or we can do like this both working.

Reply:
no worries, I also missed it 🙌